docs/declarative-customization/view-formatting.md

```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
+          "children": [
+            {
+              "elmType": "span",
+              "style": {
+                "padding": "5px 5px 5px 5px",
+                "font-weight": "500"
+              },
+              "txtContent": "@group.fieldData.displayValue"
+            }
+          ]
+        },
+        {
+          "elmType": "div",
+          "forEach": "aggregate in @aggregates",
+          "children": [
+            {
+              "elmType": "div",
+              "style": {
+                "display": "=if([$aggregate.columnDisplayName] == 'Approved' && Number([$aggregate.value]) < @group.count, 'flex', 'none')",
+                "flex-direction": "row",
+                "justify-content": "center"
+              },
+              "children": [
+                {
+                  "elmType": "div",
+                  "txtContent": "='has approval pending for ' + Number(@group.count - Number([$aggregate.value])) + if(@group.count - Number([$aggregate.value]) > 1 , ' employees', ' employee')",
+                  "style": {
+                    "padding": " ",
+                    "font-weight": "500"
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+
+```
+
+#### Example: Custom group footer
+
+In the example below we have list with group footer formatted according to aggregate value.
+![Employee list grouped by City with formatted group footer](../images/group-footer-list-layout.png)
+
+In this example the `footerFormatter` for `groupProps` is set to format the group footer and the `@columnAggregate` is used to access column aggregate.
+
+```JSON
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/sp/v2/row-formatting.schema.json",
+  "groupProps": {
+    "hideFooter": false,
+    "footerFormatter": {
+      "elmType": "div",
+      "children": [
+        {
+          "elmType": "div",
+          "attributes": {
+            "iconName": "=if(@columnAggregate.type == 'Average' && @columnAggregate.value < 10, 'SortDown', 'SortUp')"
+          },
+          "style": {
+            "color": "=if(@columnAggregate.type == 'Average' && @columnAggregate.value < 10, '#d13438', '#107c10')",
+            "font-weight": "600",
+            "margin-top": "10px"
+          }
+        },
+        {
+          "elmType": "div",
+          "style": {
+            "color": "=if(@columnAggregate.type == 'Average' && @columnAggregate.value < 10, '#d13438', '#107c10')",
+            "font-weight": "600",
+            "margin-top": "10px",
+            "font-family": "Segoe UI"
+          },
+          "txtContent": "=@columnAggregate.type + ': ' + @columnAggregate.value"
+        }
+      ]
+    }
+  }
+}
+```
+
+### Build custom list footers
+
+You can use `footerFormatter` to format list footer with access to column aggregates.
+
+#### Example: Custom list footer
+
+In the example below we have list with footer formatted according to aggregate value.

```

Comment:
1) Line equivalent space 
2) In the example below we have list with formatted footer as per the aggregate value.